test/Examples.tests.ts

```diff
@@ -0,0 +1,509 @@
+import Trace from '../src/Trace';
+import { Assembler, VM, default as whynot } from '../src/index';
```

Comment:
These test could also run from a 'built' version. This would guarantee that typescript won't trigger any bugs on our code,